classes/ReplyHandler.js

```diff
@@ -66,54 +67,16 @@ module.exports = class ReplyHandler {
 		}
 	}
 
-	/**
-	 * Replies with an error message.
-	 * @param {string} msg The message to be used.
-	 * @param {{title?: string, footer?: string, thumbnail?: string, additionalEmbeds?: EmbedBuilder[], components?: import('discord.js').ActionRowBuilder[]}} [embedExtras] Extra data to be passed to the embed.
-	 * @returns {Promise<import('discord.js').Message>|Promise<import('discord-api-types/v10').APIMessage>|boolean} The message that was sent.
-	 */
-	async error(msg, embedExtras) {
-		const replyData = this.replyDataConstructor(msg, embedExtras, true);
-		if (!this.interaction.replied && !this.interaction.deferred) {
-			replyData.ephemeral = true;
-			try {
-				return await this.interaction.reply(replyData);
-			}
-			catch (err) {
-				logger.error({ message: `${err.message}\n${err.stack}`, label: 'Quaver' });
-				return false;
-			}
-		}
-		try {
-			return await this.interaction.editReply(replyData);
-		}
-		catch (err) {
-			logger.error({ message: `${err.message}\n${err.stack}`, label: 'Quaver' });
-			return false;
-		}
-	}
-
 	/**
 	 * Replies with a localized message.
 	 * @param {string} code The code of the locale string to be used.
 	 * @param {{title?: string, footer?: string, thumbnail?: string, additionalEmbeds?: EmbedBuilder[], components?: import('discord.js').ActionRowBuilder[]}} [embedExtras] Extra data to be passed to the embed.
+	 * @param {"success"|"neutral"|"warning"|"error"} [type=neutral] Type of the message.
```

Comment:
Remove doc `@param` `[type=neutral]`, this method's parameter `type` has no default value.